sklearn/preprocessing/_function_transformer.py

```diff
@@ -176,5 +176,10 @@ def _transform(self, X, func=None, kw_args=None):
 
         return func(X, **(kw_args if kw_args else {}))
 
+    @property
+    def __sk_is_fitted__(self):
+        """Return True since FunctionTransfomer is stateless."""
+        return True
```

Comment:
Does it really need to be a property? It could just be `__sk_is_fitted = True` as a class attribute.